tests/daemon/test-persistent-cache.c

```diff
@@ -1179,7 +1127,7 @@ test_persistent_cache_get_offset_can_build_boot_metadata_file (gboolean     *unu
    * always_update_timestamps parameter is FALSE.
```

Comment:
The comment above that says "Don't write a default boot offset file [...]" should be replaced with:
```g_assert_cmpint (g_unlink (TEST_DIRECTORY BOOT_OFFSET_METADATA_FILE), ==, 0);``` (pre-existing defect).